<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBDTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Game">
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="gameVerticalLayout" stretch="3,5,1">
          <item>
           <layout class="QHBoxLayout" name="victoryHorizontalLayout" stretch="0,0,0">
            <item>
             <widget class="QGroupBox" name="victoryType">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Victory Type</string>
              </property>
              <widget class="QRadioButton" name="killerWonRadio">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>151</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="text">
                <string>Killer Won</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">victoryTypeButtonGroup</string>
               </attribute>
              </widget>
              <widget class="QRadioButton" name="hatchEscapeRadio">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>60</y>
                 <width>161</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="text">
                <string>Hatch Escape</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">victoryTypeButtonGroup</string>
               </attribute>
              </widget>
              <widget class="QRadioButton" name="gateEscapeRadio">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>90</y>
                 <width>161</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="text">
                <string>Gate Escape</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">victoryTypeButtonGroup</string>
               </attribute>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Pips</string>
              </property>
              <widget class="QRadioButton" name="pip1">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>80</y>
                 <width>51</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>+1</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="pip0">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>51</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
              <widget class="QRadioButton" name="pip2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>110</y>
                 <width>51</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>+2</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="logoLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>500</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="selectionHorizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="mapVerticalLayout">
              <item>
               <widget class="QLabel" name="selectedMapLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Not Selected</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="selectMapButton">
                <property name="text">
                 <string>Select Map</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selectedMapImgLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="killerVerticalLayout">
              <item>
               <widget class="QLabel" name="selectedKillerLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Not Selected</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="selectKillerButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Select Killer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selectedKillerImgLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="survivorVerticalLayout">
              <item>
               <widget class="QLabel" name="selectedSurviorLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Not Selected</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="selectSurvivorButton">
                <property name="text">
                 <string>Select Survivor</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selectedSurvivorImgLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="controlHorizontalLayout" stretch="0,0">
            <item>
             <spacer name="saveClearHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
              <item>
               <widget class="QLabel" name="errorLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="saveButton">
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="clearButton">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Stats">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
       <widget class="QLabel" name="bestMapLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Best Map:</string>
        </property>
       </widget>
       <widget class="QLabel" name="worstMapLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Worst Map:</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="gameHistoryScroll">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>751</width>
          <height>421</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>749</width>
           <height>419</height>
          </rect>
         </property>
         <widget class="QListWidget" name="gameHistoryList">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>751</width>
            <height>421</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="worstKillerLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Worst Killer (vs):</string>
        </property>
       </widget>
       <widget class="QLabel" name="bestKillerLabel">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Best Killer (vs):</string>
        </property>
       </widget>
       <widget class="QLabel" name="worstSuvivorLabel">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>60</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Best Survivor:</string>
        </property>
       </widget>
       <widget class="QLabel" name="bestSurvivorLabel">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Best Survivor:</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>-10</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,1,1,2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="KillerFilter">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="SurvivorFilter">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MapFilter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="victoryTypeButtonGroup"/>
 </buttongroups>
</ui>
